packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/OnLayoutEvent.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.uimanager
+
+import androidx.core.util.Pools.SynchronizedPool
+import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.WritableMap
+import com.facebook.react.common.annotations.VisibleForTesting
+import com.facebook.react.common.annotations.internal.LegacyArchitecture
+import com.facebook.react.common.annotations.internal.LegacyArchitectureLogLevel
+import com.facebook.react.common.annotations.internal.LegacyArchitectureLogger
+import com.facebook.react.uimanager.PixelUtil.toDIPFromPixel
+import com.facebook.react.uimanager.events.Event
+
+/** Event used to notify JS component about changes of its position or dimensions. */
+@LegacyArchitecture
+public class OnLayoutEvent private constructor() :
+  Event<OnLayoutEvent>() {
+  @VisibleForTesting public var x: Int = 0
+  @VisibleForTesting public var y: Int = 0
+  @VisibleForTesting public var width: Int = 0
+  @VisibleForTesting public var height: Int = 0
```

Comment:
This is confusing. 

Normally if something is marked as `VisibleForTesting`, you should just make it `internal`. Can you look into it?

Reply:
I see this was addressed now but thanks for noting this, will take it into account for other PRs.

One question: I see the annotation was left but also the `internal` keyword. Did you mean for it to be like that, or only as:

```kt
internal var width: Int = 0
```